src/vconsole/vconsole-setup.c

```diff
@@ -295,6 +302,7 @@ int main(int argc, char **argv) {
         /* Let the kernel command line override /etc/vconsole.conf */
         if (detect_container() <= 0) {
                 r = parse_env_file("/proc/cmdline", WHITESPACE,
+                                   "vconsole.n.setup.vts", &n_setup_vts_s,
                                    "vconsole.keymap", &vc_keymap,
                                    "vconsole.keymap.toggle", &vc_keymap_toggle,
```

Comment:
maybe we should have called this "vconsole.keymap_toggle" really..., but that's a different discussion...